lib/galaxy/util/__init__.py

```diff
@@ -294,13 +294,22 @@ def unique_id(KEY_SIZE=128):
     return md5(random_bits).hexdigest()
 
 
-def parse_xml(fname: StrPath, strip_whitespace=True, remove_comments=True) -> ElementTree:
+def parse_xml(
+    fname: StrPath, schemafname: Union[StrPath, None] = None, strip_whitespace=True, remove_comments=True
```

Comment:
This could break the use of this method by third-party apps (via https://pypi.org/project/galaxy-util/ ), it would be safer to have `schemafname` be added as the last parameter).

Reply:
Do you want to open a followup or should I do it ?

Reply:
I can do it later.